salsah/src/public/js/jquery.searchbox.js

```diff
@@ -206,7 +208,12 @@
 							left: pos.left,
 							top: pos.top
 						}).show();
-						$this.val('');
+						// if the input already has a value and it is valid, back it up
```

Comment:
> finding that value was not obvious to me

neither for me :-)

I think as long as the user does not select a new resource (click), the Iri remains saved so it should be ok.